src/main/riddl/Tenant/tenantProjections.riddl

```diff
@@ -1,31 +1,27 @@
 projection TenantViews is {
     fields {
-        orgId: TenantId,
-        primaryContact: Contact,
-        members: MemberId*,
-        organizations: OrganizationId*
+        tenant: TenantId,
+        member: MemberId
     }
     handler TenantViewsHandler is {
-        on event TenantEstablished {???
-            //then set ^^.TenantViews.orgId to @TenantContext.TenantEstablished.orgId
-            //and set ^^.TenantViews.primaryContact to @TenantContext.TenantEstablished.info.primaryContact
-            //and set ^^.TenantViews.members to @TenantContext.TenantEstablished.info.members
-            //and set ^^.TenantViews.organizations to @TenantContext.TenantEstablished.info.organizations
+        on event TenantEstablished {
+            then "set tenant to TenantEstablished.tenantId"
+            and "set member to TenantEstablished.info.members"
```

Comment:
I am debating whether Tenant should have Members. I think this may be a leftover from the idea of a Root Organization (which tenant evolved from). Right now, I am leaning to no.